src/keyboard/behavior-short-names.json

```
@@ -0,0 +1,14 @@
+{
+    "Key Press": {"short": ""},
+    "Bootloader": {"short": "Bootloadr"},
+    "External Power": {"short": "Ext Pwr"},
+    "Grave/Escape": {"short": "Grv/Esc"},
+    "Key Repeat": {"short": "Key Rept"},
+    "Key Toggle": {"short": "Key Togg"},
+    "Momentary Layer": {"short": "MLayer"},
+    "Output Selection": {"short": "OutputSel"},
+    "Sticky Key": {"short": "Stcky Key"},
+    "Studio Unlock": {"short": "Unlock"},
+    "Toggle Layer": {"short": "Togg Layr"},
+    "Transparent": {"short": "Trans"}
```

Sorry, last nit: Since we've got room, maybe this could be:

```suggestion
    "Transparent": {"short": "Transpnt"}
```

---

How about we drop all the vowels?
`Trnsprnt`
We also have room for
`Transprnt`
if we go by the 9 character max

---

WFM.